include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -145,15 +145,15 @@ class __radix_sort_reorder_kernel;
 //-----------------------------------------------------------------------
 
 template <typename _KernelName, ::std::uint32_t __radix_bits, bool __is_ascending, typename _ExecutionPolicy,
-          typename _ValRange, typename _CountBuf, typename _Proj
+          typename _ValRange, typename _CountBuf, typename _Event, typename _Proj
 #if _ONEDPL_COMPILE_KERNEL
           , typename _Kernel
 #endif
           >
-sycl::event
+auto
```

Comment:
I think it may be better for code maintainability to use `__dpl_sycl::__event` as the return type rather than `auto`.  (same for other similar functions in thie PR)

For the declarations, I don't mind the use of `auto`, but for returns of large functions, this seems like unnecessary obfuscation when the type is known and has a simple name.